docs/CHANGELOG.mdx

```diff
@@ -2896,7 +2896,7 @@ This is `3.12.8` release with internal infrastructure fixes to publish the docke
 ### Fixed
 
 - The `/.auth/saml/metadata` endpoint has been fixed. Previously it panicked if no encryption key was set.
-- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.11.1` will need to manually modify their `docker run` command to use `sourcegraph/server:3.11.4` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
+- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:5.3.0` will need to manually modify their `docker run` command to use `sourcegraph/server:5.3.0` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
```

Comment:
This change looks incorrect.

```suggestion
- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.11.1` will need to manually modify their `docker run` command to use `sourcegraph/server:3.11.4` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
```

@BolajiOlajide any idea how to stop the release tool from touching this file?

Reply:
Just pushed this to stop that: https://github.com/sourcegraph/sourcegraph/pull/60490

The second is a bit tricky, maybe?